schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+  id INTEGER,
```

Comment:
[OPTIONAL] Please make the `id` column `not null` and `autoincremented` because when the new table is created the id will be incremented automatically :slightly_smiling_face: 
```sql
id int NOT NULL IDENTITY(1,1)
```

[OPTIONAL] Please mark the `id` column as the `primary key` because in every table id is unique so kindly mark it the primary key :slightly_smiling_face: 
```sql
PRIMARY KEY (id)
```